src/spec/context.cr

```diff
@@ -277,7 +277,8 @@ module Spec
 
     def initialize(@parent : Context, @description : String,
                    @file : String, @line : Int32, @end_line : Int32,
-                   @focus : Bool)
+                   @focus : Bool, tags)
```

Comment:
Maybe this should provide a default value `tags = nil`? Ditto for `Example`.

Reply:
I could do that.  I hadn't done it initially because `@focus` doesn't have a default, which is a similar type of parameter.

Reply:
Yeah, they could perhaps both get default values. It's not important though.

Reply:
Let's hold off for a separate PR then.  I'm thinking there are a lot of quality of life changes around these parameters that you and I have have spoken of elsewhere, like enforcing named parameters, for example.